### DISCUSSION TITLE
Changing log levels at runtime

### DISCUSSION THREAD

**Discussion by cong-or** [tokio-rs/tracing#2053] [Q&A] [answered]
Is it possible to to dynamically change the log level at runtime?

**Comment by hawkw** [comment, marked as answer]
Yes. If you're using `tracing-subscriber`, see the [`reload` module](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/reload/index.html), which can be used for dynamically changing log levels.

**Reply by cong-or** [reply]
thanks
